tests/serverpod_test_server/lib/src/protocol_custom_classes.dart

```diff
@@ -0,0 +1,38 @@
+// ignore: depend_on_referenced_packages
+import 'package:serverpod_client/serverpod_client.dart';
+
+class ProtocolCustomClass extends SerializableEntity
```

Comment:
Suggestion: Remove extention on `SerializableEntity`, this is not required for a custom class and we don't need to validate that in this test.

Reply:
Same for the class implementation on the client side.

Reply:
In order to remove SerializableEntity from custom classes we need to update serverpod streaming, otherwise it is not possible to do this because all models/classes are required to extend SerializableEntity for serverpod streaming.